.github/workflows/labeler.yml

```diff
@@ -10,6 +10,22 @@ jobs:
     - name: Checkout
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
 
+    - name: Fetch main branch
+      run: git fetch origin main:main
+
+    - name: List local branches
+      run: git branch
```

Comment:
Why are these steps needed?

Reply:
I think this can be simplified by doing:

```
- name: Checkout
      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
      with:
        fetch-depth: 0
```

Reply:
@siggerzz just curious if using that action is really beneficial? It seems like it adds a dependency that presumably might need to be updated from time to time whereas the commands Fernando added will never change

Reply:
@siggerzz I think what Fernando has is good because we don't actually need to check out the main branch, we're just using it to compare to the current branch, unless I'm misunderstanding what the action does

Reply:
Ah, that's totally valid! I thought we were already using checkout, but I misread. My bad!